integration_tests/protractor.conf.ts

```diff
@@ -54,12 +56,12 @@ export const config: Config = {
         'password_manager_enabled': false,
       },
     },
-/**
- * 'browserName': 'firefox',
- * 'moz:firefoxOptions': {
- *   'args': ['--safe-mode']
- *   }
- */
+/*
```

Comment:
Shouldn't have commented-out code. I think it's safe to not expect Firefox testing for now.

Reply:
OK, I will remove firefox